docs/content_model.md

```diff
@@ -89,26 +89,27 @@ They represent three categories of Content Types that you can create
 
 ### Content browser
 
-If you want to get an overview of the content currently in your website, go to the Content tab
+If you want to get an overview of the content currently in your website, go to the **Content** tab
 and select **Content structure** or **Media**.
-A **Browse** option on the left enables you to browse through all the content of the respective category.
-User accounts, which are also considered content in eZ Platform, you can view through the Admin Panel
-(see [Users](organizing_the_site.md#users)).
+The **Browse** option on the left allows you to browse through all the content of the respective category.
 
 ![Content browser in the menu](img/left_menu_browse.png "Content browser in the menu")
 
+User accounts, which are also considered content in eZ Platform, are viewable in the **Users** tab accessible in the **Admin Panel** (see [Users](organizing_the_site.md#users)).
+
 During your work with eZ Platform you will often be asked to select content from the repository.
-This happens for example when you want to move or copy, a Content item, embed an image, link two Content items, etc.
+This happens for example when you want to move or copy a Content item, embed an image, link two Content items, etc.
 In such cases you will make use of the **Content Browser**.
 
-The Content Browser enables you to navigate through all content in the site, including regular content, media and User accounts.
+The **Content Browser** allows you to navigate through all content in the site, including regular content, media, and User accounts.
```

Comment:
We prefer to use "enables" in this context, while "allows" is reserved for talking about permissions in the system.